### PR TITLE
[Cosmos DB] `az cosmosdb create/update`: Add support for minimum allowed TLS version and burst capacity configuration

### PR DESCRIPTION
**Pull request by AdrianSibajaRetana**
**Related command**
az cosmosdb create

**Description**
Removed preview parameters and added test for new property.

**Testing Guide**
Added test for CMK status.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py [hunk 1]
```diff
@@ -115,12 +115,16 @@ def load_arguments(self, _):
...
             c.argument('analytical_storage_schema_type', options_list=['--analytical-storage-schema-type', '--as-schema'], arg_type=get_enum_type(AnalyticalStorageSchemaType), help="Schema type for analytical storage.", arg_group='Analytical Storage Configuration')
             c.argument('backup_policy_type', arg_type=get_enum_type(BackupPolicyType), help="The type of backup policy of the account to create", arg_group='Backup Policy')
             c.argument('enable_partition_merge', arg_type=get_three_state_flag(), help="Flag to enable partition merge on the account.")
             c.argument('continuous_tier', arg_type=get_enum_type(ContinuousTier), help="The tier of Continuous backup", arg_group='Backup Policy')
+            c.argument('minimal_tls_version', arg_type=get_enum_type(MinimalTlsVersion), help="Indicates the minimum allowed TLS version")
 
+    for scope in ['cosmosdb update']:
+        c.argument('key_uri', help="The URI of the key vault", is_preview=True)
```
Comment (rsharm7) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py:126]: is_preview?

**Comment by yonzhan** [comment]
Please fix CI issues.

**Comment by AdrianSibajaRetana** [comment]
@calvinhzy, @evelyn-ys, @jsntcy, @kairu-ms, @necusjz, and/or @wangzelin007. Can any of you folks review the changes as well please?

**Comment by pjohari-ms** [comment]
@evelyn-ys Can you review and merge this PR? Thanks.

**Review by evelyn-ys — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Pull request merged**